src/hazelcore/VarMap.re

```diff
@@ -1,7 +1,8 @@
 open Sexplib.Std;
 
 [@deriving sexp]
-type t_('a) = list((Var.t, 'a));
+type t_('a) = list(t__('a))
+and t__('a) = (Var.t, 'a);
```

Comment:
It comes from [this commit](https://github.com/hazelgrove/hazel/commit/a8afd66a36a6070cc8e8d87a212e0ef344f172b8#diff-df6427418ded9550af74f091c04f5ae6d757004913c60d472543cab543e5f922), and it seems that I wanted to avoid rewriting `(Var.t, DHExp.t)` too many times in `EvalEnv.rei`. As you point out it's not used and it's probably not very useful